rules/internal/utils/extension.go

```diff
@@ -65,6 +65,26 @@ func GetResponseType(m *desc.MethodDescriptor) *desc.MessageDescriptor {
 	return typ
 }
 
+// GetOutputOrLROResponseMessage returns the OutputType if the response is
+// not an LRO, or the ResponseType otherwise.
+func GetOutputOrLROResponseMessage(m *desc.MethodDescriptor) *desc.MessageDescriptor {
+	if m == nil {
+		return nil
+	}
+
+	if m.GetOutputType().GetName() != "Operation" {
+		return m.GetOutputType()
+	}
+
+	info := GetOperationInfo(m)
+	if info == nil {
+		return nil
+	}
+	typ := FindMessage(m.GetFile(), info.GetResponseType())
```

Comment:
I would leverage the existing `GetResponseType` to do the LRO check/`operation_info` bit instead of hand rolling it. If it returns nil, its not an LRO.

Reply:
sounds good!